src/main/java/org/jabref/gui/journals/AbbreviateAction.java

```diff
@@ -22,6 +28,8 @@
     private String message = "";
     private final boolean iso;
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbbreviateAction.class);
```

Comment:
Travis fails becoaseu of checkstyle:

```
:checkstyleMain[ant:checkstyle] [ERROR] /home/travis/build/JabRef/jabref/src/main/java/org/jabref/gui/journals/AbbreviateAction.java:18: 'org.slf4j.Logger' should be separated from previous imports. [ImportOrder]
[ant:checkstyle] [ERROR] /home/travis/build/JabRef/jabref/src/main/java/org/jabref/gui/journals/AbbreviateAction.java:31:5: Static variable definition in wrong order. [DeclarationOrder]
```

Could you please fix that, too? -- Just move it before the other variable declarations.